code/modules/tgui/states/not_incapacitated.dm

```diff
@@ -0,0 +1,29 @@
+ /**
+  * tgui state: not_incapacitated_state
+  *
+  * Checks that the user isn't incapacitated
+ **/
+
+GLOBAL_DATUM_INIT(tgui_not_incapacitated_state, /datum/tgui_state/not_incapacitated_state, new)
+
+ /**
+  * tgui state: not_incapacitated_turf_state
+  *
+  * Checks that the user isn't incapacitated and that their loc is a turf
+ **/
+
+GLOBAL_DATUM_INIT(tgui_not_incapacitated_turf_state, /datum/tgui_state/not_incapacitated_state, new(no_turfs = TRUE))
+
+/datum/tgui_state/not_incapacitated_state
+	var/turf_check = FALSE
+
+/datum/tgui_state/not_incapacitated_state/New(loc, no_turfs = FALSE)
+	..()
+	turf_check = no_turfs
+
+/datum/tgui_state/not_incapacitated_state/can_use_topic(src_object, mob/user)
+	if(user.stat)
+		return STATUS_CLOSE
+	if(user.incapacitated() || (turf_check && !isturf(user.loc)))
+		return STATUS_DISABLED
```

Comment:
need to do the lying check. They use mobility flags, but we use just plain old `lying`

Reply:
Will do

Reply:
`incapacitated()` already checks for `lying`
```dm
/mob/living/incapacitated(ignore_restraints = FALSE, ignore_grab = FALSE, ignore_lying = FALSE, list/extra_checks = list(), use_default_checks = TRUE)
	// By default, checks for weakness and stunned get added to the extra_checks list.
	// Setting `use_default_checks` to FALSE means that you don't want it checking for these statuses or you are supplying your own checks.
	if(use_default_checks)
		extra_checks += CALLBACK(src, /mob.proc/IsWeakened)
		extra_checks += CALLBACK(src, /mob.proc/IsStunned)

	if(stat || paralysis || (!ignore_restraints && restrained()) || (!ignore_lying && lying) || check_for_true_callbacks(extra_checks))
		return TRUE
```

Also seems to function as intended
![image](https://user-images.githubusercontent.com/25063394/86050105-3142db00-ba4b-11ea-89ad-4c9df4ffe6d3.png)

Reply:
Yeah, `incapacitated` checks for lying, so having the additional check isn't necessary.